Sources/Model/GraphQL/Query/HyperwalletTransferMethodConfigurationQueries.swift

```diff
@@ -209,3 +192,71 @@ public struct HyperwalletTransferMethodConfigurationKeysQuery: GraphQlQuery {
         String(format: query, userToken, limit)
     }
 }
+
+/// The 'HyperwalletTransferMethodConfigurationFeeAndProcessingTimeQuery' struct defines
+/// and builds a query to retrieve the processing time and fees associated with each country, currency
+/// and transfer method type tuple.
+/// that is required to construct a `HyperwalletTransferMethodConfigurationFieldQuery`.
+// swiftlint:disable type_name
+public struct HyperwalletTransferMethodConfigurationFeeAndProcessingTimeQuery: GraphQlQuery, Hashable {
+    private var country: String = ""
+    private var currency: String = ""
+
+    private var query = """
```

Comment:
Use this query instead `query QueryCreateTransferMethod(
    $idToken: String =  "usr-04364ef3-23c5-47b9-bf81-2fe09eb41bea",
    $country: Country = US
    $currency: Currency = USD
    $transferMethodType: TransferMethodType = BANK_ACCOUNT
) {

    processingTimes(country: $country, currency: $currency, idToken: $idToken, transferMethodType: $transferMethodType) {
        nodes {
            country
            currency
            transferMethodType
            value
        }
    }
    
    fees(country: $country, currency: $currency, idToken: $idToken, transferMethodType: $transferMethodType) {
        nodes {
            country
            currency
            transferMethodType
            feeRateType
            maximum
            minimum
            value
        }
    }
}`

Reply:
@jsaini-hw  In the above query we are sending the transfer method type. In Add Transfer Method page we need to show all the transfer methods that are applicable for the country and currency. The above query response is not convenient to parse.

Reply:
Ok